forge-gui/res/cardsfolder/upcoming/rundvelt_hordemaster.txt

```diff
@@ -0,0 +1,14 @@
+Name:Rundvelt Hordemaster
+ManaCost:1 R
+Types:Creature Goblin Warrior
+PT:1/1
+S:Mode$ Continuous | AffectedZone$ Battlefield | Affected$ Goblin.Other+YouCtrl | AddPower$ 1 | AddToughness$ 1 | Description$ Other Goblins you control get +1/+1.
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self,Goblin.Other+YouCtrl | Execute$ TrigExile | TriggerDescription$ Whenever CARDNAME or another Goblin you control dies, exile the top card of your library.
+SVar:TrigExile:DB$ Dig | DigNum$ 1 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | RememberObjects$ RememberedCard | StaticAbilities$ Play | ForgetOnMoved$ Exile | Duration$ UntilTheEndOfYourNextTurn | SubAbility$ DBCleanup
+SVar:Play:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | ConditionCheckSVar$ Z | ConditionSVarCompare$ EQ1 | Description$  If it's a Goblin creature card, you may cast that card until the end of your next turn.
```

Comment:
Maybe just move the conditions to the Effect, so it just doesn‘t get created when it‘s useless?

Reply:
Done. Should EQ1 be changed to GE1 for corner cases/future proofing?

Reply:
Should be safe I think